client/my-sites/plans/jetpack-plans/constants.ts

```diff
@@ -148,46 +147,23 @@ export const OPTION_PLAN_SECURITY: SelectorProduct = {
 				],
 			},
 			undefined,
-			getJetpackCROActiveVersion()
+			variation
 		),
 		more: MORE_FEATURES_LINK,
 	},
-};
-export const OPTION_PLAN_SECURITY_MONTHLY: SelectorProduct = {
-	...OPTION_PLAN_SECURITY,
+} );
+export const OPTION_PLAN_SECURITY_MONTHLY: ( variation: string ) => SelectorProduct = (
+	variation
+) => ( {
+	...OPTION_PLAN_SECURITY( variation ),
 	productSlug: OPTIONS_JETPACK_SECURITY_MONTHLY,
 	term: TERM_MONTHLY,
 	subtypes: [ PLAN_JETPACK_SECURITY_DAILY_MONTHLY, PLAN_JETPACK_SECURITY_REALTIME_MONTHLY ],
 	costProductSlug: PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
-};
-
-/**
- * Define properties with translatable strings getters.
- */
-[ OPTION_PLAN_SECURITY, OPTION_PLAN_SECURITY_MONTHLY ].forEach( ( target ) => {
```

Comment:
Don't we need this anymore?

Reply:
That's a good question, one that @yuliyan might be able to answer better than me. My *assumption* is that we don't need these dynamic getters anymore since the objects are being evaluated at call time, but out of caution I went back and re-added them.